css/carousel.css

```diff
@@ -0,0 +1,270 @@
+.carousel__product a:visited {
+    color: #000;
+}
+
+.carousel__wrapper {
+    width: 100%;
+    padding: 6rem 1rem;
+    position: relative;
+    overflow: hidden;
+}
+
+.carousel__wrapper:hover .carousel__button {
+    display: block;
+}
+
+/* Back and forward buttons */
+
+.carousel__button {
+    position: absolute;
+    top: 14rem;
+    background-color: #000;
+    color: #fff;
+    border-radius: 50%;
+    border: 2px;
+    padding: 5px;
+    cursor: pointer;
+    z-index: 1;
+    display: none;
+}
+
+.carousel__button:hover {
+    background-color: #2e2d2d;
+}
+
+.carousel__button > span {
+    background-color: inherit;
+    font-weight: 200;
+    font-size: 2rem;
+    border-radius: 50%;
+}
+
+#back-button {
```

Comment:
Now I see that these `back` and `forward` specific cases for `carousel__button` could be also its modifier - `carousel__button--back` etc. 😁 And as we talked they can be disabled/invisible if their action is unavailable to trigger (if carousel is on the edge)

Reply:
Hmm I thought that when using IDs I don't have to use the BEM convention, so instead I decided to use the kebab-case style. But maybe you mean that it's better to add those classes with modifiers and additionally the IDs? 
I've tried to do sth with those buttons but perhaps I need a clue to work it out...